lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/DefaultRemoteManagerImplementation.java

```diff
@@ -89,8 +88,7 @@
         numberOfTries,
         retryTimeout,
         LocalAddressProviderFactory.getInstance(),
-        RangeTcpPortProvider.Default);
-
+        new MessagingTransportFactory());
   }
 
   /**
```

Comment:
Why is this deprecated? 
It's also injected by Tang.

The deprecated annotation seems to be over-used. Please go through them and delete ones for the constructors we will keep.